components/prism-graphql.js

```diff
@@ -27,17 +37,44 @@ Prism.languages.graphql = {
 		pattern: /[a-z_]\w*(?=\s*(?:\((?:[^()"]|"(?:\\.|[^\\"\r\n])*")*\))?:)/i,
 		greedy: true
 	},
+	'constant': /\b(?!ID\b)[A-Z][A-Z_\d]*\b/,
 	'class-name': {
-		pattern: /(\b(?:enum|implements|interface|on|scalar|type|union)\s+|&\s*)[a-zA-Z_]\w*/,
-		lookbehind: true
+		pattern: /(\b(?:enum|implements|interface|on|scalar|type|union)\s+|&\s*|:\s*|\s*\[)[A-Z_]\w*/,
```

Comment:
`\s*\[`: Is this in the wrong order or is this intentional?

Assuming that it is intentional: You can remove the leading `\s*`.

```suggestion
		pattern: /(\b(?:enum|implements|interface|on|scalar|type|union)\s+|&\s*|:\s*|\[)[A-Z_]\w*/,
```

Reply:
Yes, it was intentional -- thanks for the suggestion, it works!